### DISCUSSION TITLE
Redesign App Icon

### DISCUSSION THREAD

**Discussion by IraSoro** [IraSoro/peri#283] [Q&A]
### Hello, Contributors!

We want to update our app icon and invite you to share your ideas and suggestions for the redesign.

#### What we are looking for:
We are open to various styles and concepts. We would love to hear your thoughts and see alternative versions of the icon. Here are some ideas:
- Minimalist design.
- Change in color palette.
- New symbols or abstract elements.

#### How you can contribute:
1. Leave a comment with your suggestion, concept, or sketch.
2. Attach your image and describe why you chose this particular design.

If your response is complete and fully addresses the redesign request, we will mark it as **Mark as Answer**.  

In order for your contribution to be accepted as part of Hacktoberfest, you must create a `pull request`. To do this, add the final version of the image to the `./info/icons` folder and create a `pull request`.

**If you have any questions at any step, please write! We will explain each step in detail and answer all your questions!** 

#### Description of the current icon
Now the icon looks like this:

![Image](https://github.com/user-attachments/assets/b90bc96a-41a5-4f35-b634-07d263052b09)

Please feel free to share your ideas, even if they are still in the concept stage. We welcome all participation!

**Comment by DhanushNehru** [comment]
Hi @IraSoro let me know if this icon is fine https://github.com/IraSoro/peri/issues/281

![image](https://github.com/user-attachments/assets/ee2859e5-ce40-439b-bfe4-58b31accb85d)

**Reply by IraSoro** [reply]
Hello, @DhanushNehru! Thank you for your interest in the project!

Your icon looks interesting. Can you tell me what the number 5 means?

The interface of our application has several basic colors. You can see the color palette [here](https://github.com/IraSoro/peri/blob/master/info/COLORPALETTES.md). It will be great if the icon has these colors.

**Reply by DhanushNehru** [reply]
Hi @IraSoro I just thought something like , 5 days a week. Let me know if this works, you can modify the image according to your needs if interested

![image](https://github.com/user-attachments/assets/5387e093-7768-4165-ac7c-a0072db1b83b)

**Reply by IraSoro** [reply]
@DhanushNehru, thank you for your contribution to our project! But I'm afraid this idea doesn't reflect the essence of our application. It would be great if your suggestion represent the essence of our app.

**Comment by Nabhel** [comment]
Hi @IraSoro  

Logo Concept Suggestions

I’d like to share a few thoughts regarding the new logo ideas:

1. Color Consistency: I recommend we stick with the current color scheme, unless there are plans to change the app’s overall color palette. As a user, it could feel inconsistent to log into the Peri app, which is predominantly purple, and then see a blue logo. Maintaining the existing colors will help reinforce brand identity and ensure a cohesive experience.

 2. Logo Design: It’s important to avoid straying too far from the current logo design, particularly when incorporating abstract symbols. The current symbols are intuitive and easily recognizable, which helps users quickly identify the purpose of the app. Staying close to the existing design will preserve that familiarity.

![Peri Logo 1](https://github.com/user-attachments/assets/825f41aa-57bd-401f-bc12-695de5f9ef0b)

**Reply by IraSoro** [reply]
Hello! Thanks for the work! We really appreciate it!

I like your icon options! The ideas and color combinations are really beautiful! We'll definitely save your ideas!

Unfortunately, we are not designers and don't know all the details of icon development. Therefore, we'd like to ask: if we decide to use your icon, are there any rules or recommendations that need to be followed? For example, mentioning the author, licensing or something else?

Also do you need help creating a pull request so that your contribution can be counted in Hacktoberfest? We can help with this.

**Reply by imblowfish** [reply]
Hello @Nabhel the idea is really cool and colors are beautiful as @IraSoro already mentioned

The only thing is, the drop in the upper right corner seems hardly noticeable for me if we move the logo a little further or if it will be in 64x64 or 128x128 resolutions. I'm afraid because of this, the idea behind your logo design may get lost

Your idea from the neighboring message with iterations looks more readable in these resolutions

I will note that I don't insist on changes in any way, but you clearly have an excellent vision and if you can make another iteration taking into account the readability of the icon at 64x64 and 128x128 resolutions - that would be great

**Reply by Nabhel** [reply]
@IraSoro 

Thank you for the feedback, I'm glad you liked the icon options! Regarding your questions about usage and licensing:
1. Licensing: I'd like to release the icon under the Creative Commons Attribution 4.0 International License (CC BY 4.0). This means you're free to:

- Share: Copy and redistribute the icon in any medium or format.
- Adapt: Remix, transform, and build upon the icon, even for commercial purposes.

The only requirement is attribution—you'll need to credit me as the creator. This can be done in any manner (such as in the app credits or documentation). Here’s a brief attribution you can use: (Icon designed by Nabhel and licensed under CC BY 4.0)

2. Pull Request: I’d really appreciate your help with the pull request to ensure my contribution is counted for Hacktoberfest. If you could guide me through the process, that would be great!

@imblowfish I don't totally understand what you mean, but here's what the peri logo looks like in these dimensions (64x64 , 128x128)

![Logo Dimensions](https://github.com/user-attachments/assets/56c40d71-b676-4da6-a227-4659421adb9e)

**Reply by imblowfish** [reply]
Here is what I'm talking about. I created a board in Penpot (Figma alternative) with desktop size equals to Google Pixel (393x851 pixels) an added your icons with 64x64px size

![image](https://github.com/user-attachments/assets/e4bb67cf-1320-4f13-8940-9f280c44266d)

In my opinion the drop in the first two variants look more readable in this resolution. But on the third version it's hard to see drop in the upper right corner

**Reply by Nabhel** [reply]
@imblowfish 

By hard to read, do you mean you can't see it or it's too small and you want me to make the logo bigger (i.e the blood icon).

![Make it bigger](https://github.com/user-attachments/assets/a4106331-8e66-4cfc-b702-5252f4eb6d1d)


Also do you prefer the iterations instead?

**Reply by imblowfish** [reply]
> @imblowfish
> 
> By hard to read, do you mean you can't see it or it's too small and you want me to make the logo bigger (i.e the blood icon).
> 
> ![Make it bigger](https://private-user-images.githubusercontent.com/63253907/374609214-a4106331-8e66-4cfc-b702-5252f4eb6d1d.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgzOTkzOTksIm5iZiI6MTcyODM5OTA5OSwicGF0aCI6Ii82MzI1MzkwNy8zNzQ2MDkyMTQtYTQxMDYzMzEtOGU2Ni00Y2ZjLWI3MDItNTI1MmY0ZWI2ZDFkLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA4VDE0NTEzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3ZWQzMTRmYjBkM2Q4ODhlNjEwZjQ2YjFjODZmZDY5MjIwODllYjk5MGNjNDQ2ZmE2MzZlMDc1MWJlMjVkNjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.kKSRg3XToetv3cbQAz8boVI2sJI_ms11lZZGnenjwkY)
> 
> Also do you prefer the iterations instead?

By hard to read I meant it's hard to see the drop
But this iteration with an enlarged drop is better and looks perfect, thank you

**Reply by imblowfish** [reply]
@IraSoro what do you think?

**Reply by imblowfish** [reply]
@Nabhel Here I wrote the tutorial about how to create your Pull Request using only Github Web UI https://gist.github.com/imblowfish/ab2e5663a44c3120da4ed35ed50cfac6

If you have any question or if you'll struggle on some step please feel free to ask me or @IraSoro. We'll try to help you

On our side we will add a new task so that the authorship mention will be available from the application. Me or @IraSoro will share link with you later

**Reply by IraSoro** [reply]
> @IraSoro what do you think?

I think this drop is now more visible. @Nabhel, thank you!

**Reply by Nabhel** [reply]
@imblowfish Thank you so much for the guide. I've created a Pull Request.

**Comment by Nabhel** [comment]
@IraSoro 

I also created some other versions, these are my iterations.

![Peri Logo 3](https://github.com/user-attachments/assets/077690c3-31b3-48c5-8573-b9f896129490)


![Peri Logo 2](https://github.com/user-attachments/assets/92436db6-14be-47d8-b2c4-4088903e7c0c)

**Comment by MMETehrani** [comment]
Exciting opportunity! I'll brainstorm some ideas and share my concepts soon. Looking forward to contributing

**Reply by IraSoro** [reply]
Hello, @MMETehrani! Thank you for your interest in the project! We look forward to your ideas!
